### PR TITLE
Adding Nodle to parachains

### PR DESCRIPTION
**Pull request by hobbydev71**

### CODE REVIEW

**Review by noisefloors — APPROVED** [review]

File: packages/apps-config/src/endpoints/index.spec.ts [hunk 1]
```diff
@@ -59,40 +59,6 @@ describe('urls are sorted', (): void => {
...
-
-  Object
-    .entries(map)
-    .forEach(([url, paths]) =>
-      it(url, (): void => {
-        assert(paths.length === 1, `${url} appears multiple times - ${paths.map((p) => `\n\t"${p}"`).join('')}`);
-      })
-    );
-});
-
```
Comment (aliXsed) [inline packages/apps-config/src/endpoints/index.spec.ts:95]: Why did you need to remove this test entirely?

Comment (hobbydev71) [reply packages/apps-config/src/endpoints/index.spec.ts:95]: Currently, this test is to prevent duplication of urls in the chain configuration of the app.
By the way, we need to add twice this url(wss://nodle-parachain.api.onfinality.io/public-ws).
Because we should customize the chain configuration. so I added nodle chains on top. and added that url on polkadot parachains.
As a result, duplicates with the same url.

**Review by aliXsed — APPROVED** [review]
